modules/signing_scriptworker/templates/passwords-mobile.json.erb

```diff
@@ -10,5 +10,11 @@
     ],
     "project:mobile:fenix:releng:signing:cert:release-signing": [
         ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_fenix_prod_username"]) %>", "<%= scope.function_secret(["autograph_fenix_prod_password"]) %>", ["autograph_fenix"], "autograph"]
+    ],
+    "project:mobile:reference-browser:releng:signing:cert:release-signing": [
+        ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_reference_browser_prod_username"]) %>", "<%= scope.function_secret(["autograph_reference_browser_prod_password"]) %>", ["autograph_reference_browser"], "autograph"]
+    ],
+    "project:mobile:reference-browser:staging:releng:signing:cert:release-signing": [
```

Comment:
Hm, I'm not sure this is right. If we wanted "staging" environment testing, I think using `dep-signing` certificate scope is the way to go. AFAIK, that's how we do it for the rest of the products. See https://github.com/mozilla-releng/build-puppet/blob/master/modules/signing_scriptworker/templates/dep-passwords.json.erb. 

So in this case, I'd personally create a `dep-passwords-mobile.json.erb` that is a mirror of this file but with `dep-signing` scope instead. AFAIK, even the dep counterparts are also stored on the Autograph production instances, so we'll hit that anyway. This has the advantage that we can separate nicely between production signing and "staging" signing directly from the scope. We'd need dedicated clients for this one too from SecOps.

I'm curious what @JohanLorenzo thinks as I've been ramping up in this too in the past few weeks/months so I may be wrong.

Reply:
`project:mobile:reference-browser:releng:signing:cert:dep-signing` is the way to go :)

Reply:
Moved to `dep-passwords-mobile.json.erb` and renamed the properties :+1: